pkg/operation/care/checker.go

```diff
@@ -229,31 +229,47 @@ func (b *HealthChecker) checkNodes(condition gardencorev1beta1.Condition, nodes
 
 // CheckManagedResource checks the conditions of the given managed resource and reflects the state in the returned condition.
 func (b *HealthChecker) CheckManagedResource(condition gardencorev1beta1.Condition, mr *resourcesv1alpha1.ManagedResource) *gardencorev1beta1.Condition {
+	conditionsToCheck := map[gardencorev1beta1.ConditionType]func(status gardencorev1beta1.ConditionStatus) bool{
+		resourcesv1alpha1.ResourcesApplied: defaultSuccessfulCheck(),
+		resourcesv1alpha1.ResourcesHealthy: defaultSuccessfulCheck(),
+	}
```

Comment:
For completeness we could add a test case in `health_check_test.go` for the `CheckManagedResource` func that makes sure when there is a Unknown condition, the returned condition will be `False`.

Reply:
done